src/updaterwindow.cpp

```diff
@@ -100,7 +100,7 @@ UpdaterWindow::UpdaterWindow(QWidget *parent)
     updateTitleLabel();
 
     /* Remove window border */
-    setWindowFlags(Qt::Window | Qt::CustomizeWindowHint);
```

Comment:
Would it make sense to subclass `UpdaterWindow` from `QDialog` in the first place?